packages/analytics/src/utils.ts

```diff
@@ -40,3 +40,50 @@ export const encodeDataToQueryString = <T extends Event>(
 
     return params.toString();
 };
+
+const reportEventError = (
+    type: ReportEventProps['type'],
+    retry: ReportEventProps['retry'],
+    err: any,
+) => {
+    let errorMessage = err?.error?.message || err?.message;
+
+    if (typeof errorMessage !== 'string') {
+        // this should never happen
+        errorMessage = 'Unknown error.';
+    }
+
+    // to circumvent sentry inbound filter
+    if (errorMessage.includes('Failed to fetch')) {
+        errorMessage = 'Failed to analytics fetch.';
```

Comment:
```suggestion
        errorMessage = 'Failed to fetch analytics.';
```

Reply:
It is like that to circumvent sentry inbound filter

Reply:
The message must not contain "Failed to fetch" so we could distinguish from other "Failed to fetch" errors in sentry.